src/components/instant-apps-social-share/instant-apps-social-share.tsx

```diff
@@ -331,7 +326,7 @@ export class InstantAppsSocialShare {
               <calcite-popover
                 ref={(el: HTMLCalcitePopoverElement) => (this.popoverRef = el)}
                 label={this.messages?.share?.label}
-                reference-element="shareButton"
+                referenceElement="shareButton"
```

Comment:
I believe this is still `reference-element` as the attribute, right? Or did it get changed? https://developers.arcgis.com/calcite-design-system/components/popover/#component-api-properties-referenceElement

Reply:
@Csmith246 you're correct but the typings threw an error, updating to use this casing resolved the issue and the popover still opens as expected, so i think we're good